zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -948,6 +950,15 @@ impl Parameters {
             .with_lockbox_disbursements(lockbox_disbursements.unwrap_or_default())
             .with_checkpoints(checkpoints.unwrap_or_default());
 
+        #[cfg(any(test, feature = "proptest-impl"))]
+        if let Some(should_extend_funding_stream_addresses) =
+            extend_funding_stream_addresses_as_required
+        {
+            if should_extend_funding_stream_addresses {
+                parameters = parameters.extend_funding_streams();
+            }
+        }
```

Comment:
Nitpick: Conditions could be condensed.

```suggestion
        if Some(true) == extend_funding_stream_addresses_as_required {
            parameters = parameters.extend_funding_streams();
        }
```